src/renderer/css/components/TabBar.scss

```diff
@@ -0,0 +1,202 @@
+    overflow: hidden;
+    text-overflow: ellipsis;
+    white-space: nowrap;
+    text-align: center;
+    -webkit-font-smoothing: auto;
+  }
+
+  .TabBar>li {
+    -webkit-app-region: no-drag;
+    -webkit-user-select: none;
+    min-width: 48px;
+    max-width: 224px;
+    position: relative;
+    margin: 0 2px;
+  }
+
+  .TabBar>li>a {
+    color: rgba(61,60,64,0.7);
+    font-family: "Open Sans", sans-serif;
+    font-weight: 600;
+    font-size: 12px;
+    letter-spacing: -0.2px;
+    transition: 0.3s;
+    line-height: 18px;
+    border-radius: 4px;
+    border: none !important;
+    padding: 0;
+  }
+
+  .TabBar.darkMode>li>a {
+    color: rgba(221,221,221,0.64);
+  }
+
+  .TabBar>li>a:hover {
+    background-color: rgba(255,255,255,0.56);
+    text-decoration: none;
+  }
+
+  .TabBar.darkMode>li>a:hover {
+    background-color: rgba(31, 31, 31, 0.56);
+  }
+
+  .TabBar>li>a:focus {
+    background-color: #fff;
+    color: #3d3c40;
+    text-decoration: none;
+  }
+
+  .TabBar>li.teamTabItem>a>div.TabBar-tabSeperator>.teamTabItem__close {
+      background: none;
+      border: none;
+      color: rgba(61,60,64,0.32);
+      margin-left: 6px;
+      padding: 0;
+  }
+
+  .TabBar>li.teamTabItem>a>div.TabBar-tabSeperator>.teamTabItem__close:hover {
+      color: #3d3c40;
+  }
+
+  .TabBar.darkMode>li.teamTabItem>a>div.TabBar-tabSeperator>.teamTabItem__close:hover {
+      color: #ddd;
+  }
+
+  .TabBar>li.teamTabItem>a>div.TabBar-tabSeperator>.teamTabItem__close>i::before {
+      margin: 0;
+  }
+
+  .TabBar.darkMode>li.teamTabItem>a>div.TabBar-tabSeperator>.teamTabItem__close {
+      color: rgba(221,221,221,0.32);
+  }
+
+  .TabBar.darkMode>li>a:focus {
+    background-color: #1f1f1f;
+    color: #ddd;
+  }
+
+  .TabBar>li>a>div.TabBar-tabSeperator {
+    display: flex;
+    padding: 5px 12px;
+    align-items: center;
+  }
+
+  .TabBar>li.teamTabItem.active>a, .TabBar>li.teamTabItem.dragging>a {
+    border: none;
+    background-color: #fff;
+    color: #3d3c40;
+    z-index: 9;
+  }
+
+  .TabBar.darkMode>li.teamTabItem.active>a, .TabBar.darkMode>li.teamTabItem.dragging>a {
+      background-color: #1f1f1f;
+      color: #ddd;
+  }
+
+  .TabBar>li.teamTabItem:not(.active)+li.teamTabItem:not(.active)>a>div.TabBar-tabSeperator {
+    border-left: 1px solid rgba(61,60,64,0.08);
+    margin-left: -1px;
+  }
+
+  .TabBar.darkMode>li.teamTabItem:not(.active)+li.teamTabItem:not(.active)>a>div.TabBar-tabSeperator {
+    border-left: 1px solid rgba(221,221,221,0.08);
+    margin-left: -1px;
+  }
+
+  .TabBar>li.teamTabItem:not(.active):not(.disabled):hover+.TabBar-addServerButton>a>div.TabBar-tabSeperator {
+    border-left: none;
+    margin-left: 0px;
+  }
+
+  .TabBar>li.teamTabItem:not(.active):not(.disabled):hover+li.teamTabItem:not(.active)>a>div.TabBar-tabSeperator {
+    border-left: none;
+    margin-left: 0px;
+  }
+
+  .TabBar>li.teamTabItem:not(.active):not(.disabled)+li.teamTabItem:not(.active)>a:hover>div.TabBar-tabSeperator {
```

Comment:
how about applying only one class in the case where this condition is met?

Reply:
Same here, can address in a future cleanup.